feature-utils/poly-look/src/react-components/buttons/polyButton.jsx

```diff
@@ -1,12 +1,23 @@
 import React from "react";
 import "./polyButton.css";
 
+const types = {
```

Comment:
This is a big PR already. It would be better if other changes unrelated to generating the documentation were kept off it...